creator-node/src/utils/decisionTree.ts

```diff
@@ -28,7 +30,7 @@ module.exports = class DecisionTree {
 
   tree: Stage[]
 
-  public constructor({ name, logger }: ConstructorParams) {
+  public constructor({ name, logger = genericLogger }: ConstructorParams) {
```

Comment:
i intentionally did not do this bc i wanted to force caller to pass logger wdyt

Reply:
hmmmm I think it's okay cus passing in a logger usually means we're passing in context. if there's no context, the decision tree will give us some. I think it's okay here for simpler flows that need the decision tree without the logger

Reply:
tbh i don't agree but all good it's nbd